importer-db-consistency/scripts/EpochSlotToBlkHash.js

```diff
@@ -0,0 +1,84 @@
+var fetch = require("node-fetch");
+var lodash = require("lodash");
+
+const serverRoute = 'https://explorer.iohkdev.io/api';
+
+const slotsPerEpoch = 21600;
```

Comment:
can we not read this from somewhere else in the code?

Reply:
originally from https://github.com/input-output-hk/cardano-sl/blob/2d44c0ac73c6756e4533d5e0fdee02b2e2cc3c01/core/src/Pos/Core/Configuration/Protocol.hs#L70 I think